internal/cmd/egctl/testdata/translate/out/authn-single-route-single-match-to-xds.all.yaml

```diff
@@ -236,7 +236,7 @@ xds:
                       ads: {}
                       resourceApiVersion: V3
                     routeConfigName: envoy-gateway-system/eg/http
-                  statPrefix: http
+                  statPrefix: envoy-gateway-system/eg/http
```

Comment:
I'd think it's a good pratiace to add prefix like `envoy-gateway-system/eg`.

Reply:
It may be possible to optimize the existing listener name later to provide richer indicator information. 
For example:
`{listener.gateway.Namespace}.{listener.gateway.Name}.{listener.Name}->{listener.gateway.Namespace}.{listener.gateway.Name}.{listener.Protocol}{listener.Name}`
By adding custom [stats extraction rules](https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/metrics/v3/stats.proto#envoy-v3-api-msg-config-metrics-v3-tagspecifier),we can extract the stats as `envoy_http_downstream_rq{gateway_listener_name="",Protocol="",namespace="",gateway_name=""}`

Reply:
Of course this is also possible, but if we add some additional information (the listener name of the gateway resource), we can count metrics at the listener granularity, not just the gateway granularity. This may be very useful.

Reply:
if we really need that, prefer to use metadata.

Reply:
metadata? （is metadata_exchange?）